frontend/src/homePage.js

```diff
@@ -47,16 +48,15 @@ class Home extends React.Component {
       });
   };
 
-
   handleFoldersClick = (e) => {
-    window.location = "/myFolders?userKey=" + this.state.userKey;
+    window.location = "/myFolders";
```

Comment:
This still doesn't have the same capitalization as the design doc.

Reply:
Sorry for the miscommunication but what you were expecting is /MyFolders right?

Reply:
done